source/features/parse-backticks.tsx

```diff
@@ -5,46 +5,71 @@ import * as pageDetect from 'github-url-detection';
 		'.message', // `isCommitList`, `isRepoTree`, `isBlame`
-		'.Box--condensed .link-gray[href*="/commit/"]', // `isSingleFile`
+		'.repository-content .js-details-container .link-gray[href*="/commit/"]', // `isSingleFile`
+		'.repository-content .js-details-container pre', // `isSingleFile`
 		'[aria-label="Issues"][role="group"] .js-navigation-open', // `isConversationList`
 		'[id^=ref-issue-]', // Issue references in `isIssue`, `isPRConversation`
 		'[id^=ref-pullrequest-]', // PR references in `isIssue`, `isPRConversation`
 		'.TimelineItem-body > del, .TimelineItem-body > ins', // Title edits in `isIssue`, `isPRConversation`
-		'.js-pinned-issue-list-item > span', // Pinned Issues
+		'.js-pinned-issue-list-item > .d-block', // Pinned Issues
 		'.pulse-section li', // `isPulse`
-		'.issues_labeled .text-gray-dark > a', // `isDashboard` "help wanted" event titles
-		'.js-recent-activity-container .text-bold', // `isDashboard`"Recent activity" titles
-		'.commits blockquote', // Newsfeed commits
 		'.release-header', // `isReleasesOrTags` Headers
-		'.Box-row.js-navigation-item a.link-gray-dark', // `isGlobalConversationList`
-		'[data-channel^="check_suites"] a', // `isActions`
+		'[id^="check_suite"] a.link-gray-dark', // `isActions`
 		'.repository-content .pr-toolbar h2', // `isActions` run
-		'#wiki-wrapper h1' // `isWiki`
-	].map(selector => selector + ':not(.rgh-backticks-already-parsed)'))) {
-		title.classList.add('rgh-backticks-already-parsed');
-		parseBackticks(title);
-	}
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden a.link-gray-dark', // `isRepoRoot`
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden pre', // `isRepoRoot`
+		'.Details[data-issue-and-pr-hovercards-enabled] .d-none a.link-gray-dark', // `isRepoRoot`
+		'.existing-pull-contents .list-group-item-link', // `isCompare` with existing PR
+		'[aria-label="Link issues"] a', // "Linked issues" in `isIssue`, `isPRConversation`
+		'.js-wiki-sidebar-toggle-display a', // `isWiki`
+		'.gh-header-title' // `isWiki`
+	]);
+}
+
+function dashboardInit(): void {
+	parse([
+		'.js-recent-activity-container .text-bold', // `isDashboard`"Recent activity" titles
+		'.commits blockquote' // Newsfeed commits
+	]);
+}
+
+function notificationsInit(): void {
+	parse([
+		'.notifications-list-item p.text-normal' // `isNotifications`
+	]);
 }
 
 void features.add({
 	id: __filebasename,
-	description: 'Renders text in `backticks` in issue titles and commit titles/descriptions.',
+	description: 'Renders text in `backticks` in issue titles, commit titles and more places.',
 	screenshot: 'https://user-images.githubusercontent.com/170270/55060505-31179b00-50a4-11e9-99a9-c3691ba38d66.png'
 }, {
 	include: [
 		pageDetect.isRepo
 	],
-	init
+	init: repoInit
 }, {
 	include: [
 		pageDetect.isDashboard
 	],
 	onlyAdditionalListeners: true,
-	init
+	init: dashboardInit
```

Comment:
```suggestion
	init: initDashboard
```

same for notifications